python/src/otel/otel_sdk/requirements-nodeps.txt

```
@@ -1,29 +1,29 @@
 opentelemetry-instrumentation-aiohttp-client==0.29b0
```

Does this need to be updated?

```suggestion
opentelemetry-instrumentation-aiohttp-client==0.30b1
```